main.go

```diff
@@ -129,3 +142,8 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// spacesAccessKeysSet returns if the spaces access key id and the spaces access key secret are set
+func (c *Config) spacesAccessKeysSet() bool {
+	return c.SpacesAccessKeyID != "" && c.SpacesAccessKeySecret != ""
+}
```

Comment:
I wouldn't hide this behind a helper function. You can just call this in the if twice.

Reply:
Fixed 👍